### PR TITLE
feat: updates OramaSearch data after 2.0 release

### PR DESCRIPTION
**Pull request by micheleriva**
Hi!
We just released Orama 2.0, which introduces hybrid search and a lot more features. You can read the full announcement here: https://oramasearch.com/blog/orama-2-0-is-out-hybrid-search-geosearch-secure-ai-proxy-orama-cloud-and-more.

I have updated the `oramasearch.json` field to reflect the new features of Orama 2.0.

Cheers

### CODE REVIEW

File: docs/tools/vdb_table/data/oramasearch.json [hunk 1]
```diff
@@ -25,15 +25,15 @@
...
+  "github_stars": 7100,
   "vector_launch_year": 2022,
   "metadata_filter": {
     "support": "full",
     "source_url": "",
     "comment": ""
   },
   "hybrid_search": {
-    "support": "none",
+    "support": "full",
```
Comment (dhruv-anand-aintech) [inline docs/tools/vdb_table/data/oramasearch.json:36]: please provide a reference to docs/example notebook for this in "source_url". It should show ability to merge between ranked lists of keyword search and vector search results.

File: docs/tools/vdb_table/data/oramasearch.json [hunk 2]
```diff
@@ -48,7 +48,7 @@
     "comment": ""
   },
   "multi_vec": {
-    "support": "none",
+    "support": "full",
```
Comment (dhruv-anand-aintech) [inline docs/tools/vdb_table/data/oramasearch.json:51]: please provide a reference to docs/example notebook for this in "source_url"

File: docs/tools/vdb_table/data/oramasearch.json [hunk 3]
```diff
@@ -108,24 +108,24 @@
...
   },
   "pricing": {
-    "value": "free plan / $4.99 per month (pro)",
+    "value": "free plan / $29.99 per month (pro)",
     "source_url": "",
     "comment": ""
   },
   "in_process": {
-    "support": "none",
+    "support": "full",
```
Comment (dhruv-anand-aintech) [inline docs/tools/vdb_table/data/oramasearch.json:116]: please provide a reference to docs/example notebook for this in "source_url"

File: docs/tools/vdb_table/data/oramasearch.json [hunk 4]
```diff
@@ -108,24 +108,24 @@
...
   "multi_tenancy": {
     "support": "full",
     "source_url": "",
     "comment": ""
   },
   "disk_index": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
```
Comment (dhruv-anand-aintech) [inline docs/tools/vdb_table/data/oramasearch.json:126]: This is the meaning of this column: https://github.com/superlinked/VectorHub/blob/a986982150dfce541551a3a9a424b63b98920918/docs/tools/vdb_table/vendor.schema.json#L33

The index should be served from disk to qualify

**Review by dhruv-anand-aintech — CHANGES_REQUESTED** [review]

File: docs/tools/vdb_table/data/oramasearch.json (hunk 3 above)
Comment (micheleriva) [reply docs/tools/vdb_table/data/oramasearch.json:116]: Orama is written in JavaScript, therefore you can embed it in any JS application on the browser/server. That's well stated in our docs/repo

File: docs/tools/vdb_table/data/oramasearch.json (hunk 4 above)
Comment (micheleriva) [reply docs/tools/vdb_table/data/oramasearch.json:126]: Yes, we support this by storing partitions on disk via the plugin I wrote down in the `source_url` field. Orama Cloud does that automatically for you, OSS orama requires you to perform this operation on your own

**Comment by svonava** [comment]
@micheleriva friendly ping here - could you provide the source links as per @dhruv-anand-aintech comment so we can get this merged in? Thanks!

**Review by dhruv-anand-aintech — APPROVED** [review]

**Pull request merged**